build.gradle

```diff
@@ -125,7 +125,7 @@ subprojects {
   }
 
   test {
-    maxHeapSize = "1024m"
+    maxHeapSize = "2048m"
```

Comment:
As discussed, let's try to not have to do that. Which means that most tests should properly destroy the entity when done.